elastic4s-core/src/test/scala/com/sksamuel/elastic4s/mappings/MappingDefinitionDslTest.scala

```diff
@@ -9,6 +9,26 @@ class MappingDefinitionDslTest extends WordSpec with Matchers with JsonSugar {
   import com.sksamuel.elastic4s.ElasticDsl._
 
   "mapping definition" should {
+    "insert source exclusion directives when set" in {
+      val mapping = new MappingDefinition("test").sourceExcludes("excludeMe1", "excludeMe2")
+      val output = mapping.build.string()
+      println("output = " + output)
```

Comment:
Thanks @martin-g , good spot. I've fixed it here baeb07e8eb77213b6a473c1668c693bad799116e